src/otx/algorithms/classification/configs/deit_tiny/model.py

```diff
@@ -14,7 +14,7 @@
     backbone=dict(arch="deit-tiny", init_cfg=dict(type="Pretrained", checkpoint=ckpt_url, prefix="backbone")),
 )
 
-fp16 = dict(loss_scale=512.0)
+# fp16 = dict(loss_scale=512.0)
```

Comment:
Let's make this change in train function, not in templates. Depending on the given device we can set fp16 to None (if it is XPU devices).

Reply:
Done